website/docs/reference/rbac.md

```diff
@@ -161,23 +161,39 @@ Each custom project role consists of:
 You can assign the following project permissions. The permissions will be valid across all of the project's
 environments.
 
+#### Features and strategies
+| Permission | Description | 
+| --- | --- |
+| **create feature flags within the project** | Lets the user create feature flags within the project and create variants for said flag. Note that they **cannot assign strategies** to flags without having the _create activation strategy_ permission for the corresponding environment. |
+| **update feature flags within the project** | Lets the user update feature flag descriptions; mark flags as stale / not stale; add, update, and remove flag tags; and update flag variants within the project.                                                                          |
```

Comment:
Could we add Update feature dependency and Create/edit project segment too please